config/webpack.config.js

```diff
@@ -50,7 +50,7 @@ var config = {
   resolve: {
     modules: [
       path.join(__dirname, '..', 'webpack'),
-      path.join(__dirname, '..', 'node_modules')
+      'node_modules/'
```

Comment:
@ohadlevy I'm not sure as I don't have any plugin with npm dependencies, It does break Foreman nightlies or even development if installed with `--global-style true` though. 

If you're worried about removing the line, I think it should be fine to leave it at 
```javascript
path.join(__dirname, '..', 'node_modules'),
'node_modules/'
```
 and have both lines. However this would also NOT work with plugins that have nested dependencies as it'd only look in the first level deps.